snailmail.py

```diff
@@ -116,9 +116,6 @@ def make_mail(sender, recipient, subject, body, attachments):
         if input("Abort? [y/N] ").lower() == "y":
             sys.exit(0)
 
-    username = args.username or input("Username: ")
```

Comment:
Please keep this around and add a comment "(leave blank for unauthenticated login)"